ricecooker/classes/nodes.py

```diff
@@ -148,6 +150,9 @@ class ChannelNode(Node):
             title (str): name of channel
             description (str): description of the channel (optional)
             thumbnail (str): file path or url of channel's thumbnail (optional)
+            license (str): default license to use if nodes don't have license specified (optional)
```

Comment:
We now require license to be specified on each node, so channel-level license can be removed.